### DISCUSSION TITLE
reload.js:96 WebSocket connection to 'ws://my_server_ip:38779/' failed,reported in browser ervery minute

### DISCUSSION THREAD

**Discussion by 13316090972** [lowdefy/lowdefy#1405] [Questions and Help] [answered]
I run lowdefy app in develop cli in my server.
Though I can use the lowdefy app ,the error like title reported in browser developer tools every minute .
It makes me somehow uncomfortable,any way can resolve it?
![image](https://user-images.githubusercontent.com/56707831/198478587-e40f4c92-5e11-47b1-9f8d-130b052e3ae4.png)

**Comment by 13316090972** [comment]
The server  only  enable exactly ports ,like 3000.
The websocket port 38779 seems to be flexable,and will be changed when I run `lowdefy dev` next time.

**Comment by SamTolmay** [comment, marked as answer]
Hi @13316090972 

The dev server opens a websocket connection for the reload functionality, where it reloads if changes are made to your configuration. This is opened on a random port, with some logic to find an open port in the ~30 000 range. The reason we need to find a random is because the you might be running more than one server, and we didn't want to add additional configuration for users that want to run two apps at the same time.

This means that there is no issue if this is blocked, you will just be missing out on the reload functionality. I don't know what to do about the error, maybe you can hide that specific error using chrome dev tools.

In V4 this won't be an issue, since the reload runs  on the same port as the server
